### PR TITLE
Add Claude icon

### LINKED ISSUE #10974: Request: Claude
### Brand Name

Claude

### Website

https://claude.ai

### Popularity Metric

The similar web rank is 927. See https://www.similarweb.com/website/claude.ai/#overview

### Official Resources for Icon and Color

PNG: https://claude.ai/images/claude_app_icon.png from the website header

### Additional Comments

Although the source I've provided is a PNG,on their home page, they've used the SVG code directly, so I think we can extract the SVG code to build the SVG icon by removing the letter path.

### PR DESCRIPTION
**Pull request by uncenter**
claude

**Issue:** closes #10974.

**Popularity metric:** Ranked 927 globally on Similarweb. See https://www.similarweb.com/website/claude.ai/#overview.

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: icons/claude.svg
Comment (adamrusted) [inline icons/claude.svg:1]: The path for this one has _way_ too many excess points, can you see if you can reduce them at all @uncenter?
image

Comment (uncenter) [reply icons/claude.svg:1]: Hmmm not many of those are truly excess and have an actual affect on the shape. The most I can take off is about one/two from each "arm" of the shape but that too requires fine tuning to keep it lined up with the original.

Comment (adamrusted) [reply icons/claude.svg:1]: Fair enough - let's see if @PeterShaggyNoble can work his magic? 😅

**Review by mondeja — APPROVED** [review]
I can confirm that the excess points are part of the shape. Looks fine to me!

**Pull request merged**